### PR TITLE
Grab background images no larger than the screen

### PR DESCRIPTION
**Pull request by DoggoOfSpeed**
**Changes**
Fix the app grabbing 4k background images even when the screen is just 1080p.

As a side-effect that also saves a lot of cache space :)

### CODE REVIEW

**Comment by nielsvanvelzen** [comment]
I'm not sure about this one. Yes it reduces the image size on smaller screens which leads to smaller cache but at the same time it will now make the server resize all images and cache them there which makes the (first) requests slower.

**Comment by DoggoOfSpeed** [comment]
Well for me personally that still makes some sense.

Sure, it takes time to process the images, but in turn that means the images require less bandwith and are quicker to download.

The same can kinda be applied to storage space as the server, in all likelyhood, is much less storage limited whereas many tvs and set-top boxes don't even have 8gb.

**Comment by nielsvanvelzen** [comment]
Right that's true, should be fine to change this then.

File: app/src/main/java/org/jellyfin/androidtv/data/service/BackgroundService.kt [hunk 1]
```diff
@@ -60,6 +61,7 @@ class BackgroundService(
 				imageType = ImageType.BACKDROP,
 				tag = tag,
 				imageIndex = index,
+				maxWidth = Resources.getSystem().getDisplayMetrics().widthPixels,
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/data/service/BackgroundService.kt:64]: I believe we can use `context.resources` here

Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/androidtv/data/service/BackgroundService.kt:64]: Yeah you should be able to do `context.resources.displayMetrics.widthPixels`

**Comment by nielsvanvelzen** [comment]
Looking a bit further we should use `fillWidth` instead of `maxWidth` and also add `fillHeight`.

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**